business/dao/users-mysql-dao.js

```diff
@@ -56,9 +56,23 @@ function getUserIdFromUuid(uuid, cb) {
 
 }
 
+function getUserTasks(id, cb) {
+	pool.getConnection(function onConnection(err, connection) {
+		if(err) {
+			cb(err);
+		} else {
+			connection.query('select t.*, ut.priority from users_tasks ut JOIN tasks t ON ut.task_id = t.id where ut.user_id = ?', [id], function(err, results) {
+                connection.release();
+                cb(err, results);
+			});
+		}
+	});
+}
+
 module.exports = {
 	'createUser': createUser,
 	'getUserById': getUserById,
 	'getUserIdFromUuid': getUserIdFromUuid,
-	'getUserByUuid': getUserByUuid
+	'getUserByUuid': getUserByUuid,
+    'getUserTasks': getUserTasks
```

Comment:
We're basically going to need to refactor all of this, instead of reviewing
this we could just update the schema and I'll fix all of this stuff

On Sun, Aug 9, 2015 at 1:08 PM, jmccarthy14 <notifications@github.com>
wrote:

> In business/dao/users-mysql-dao.js
> <https://github.com/jmccarthy14/perfectpm-backend/pull/1#discussion_r36593063>
> :
>
> >  module.exports = {
> >  	'createUser': createUser,
> >  	'getUserById': getUserById,
> >  	'getUserIdFromUuid': getUserIdFromUuid,
> > -	'getUserByUuid': getUserByUuid
> > +	'getUserByUuid': getUserByUuid,
> > +    'getUserTasks': getUserTasks
>
> tabs
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jmccarthy14/perfectpm-backend/pull/1/files#r36593063>.
>